app/policies/form_policy.rb

```diff
@@ -45,7 +49,11 @@ def can_edit_page_routing_conditions?
 
 private
 
+  def user_is_form_creator
+    form.respond_to?(:creator_id) ? user.id == form.creator_id : false
+  end
+
   def users_organisation_owns_form
-    user.organisation.slug == form.org
+    user.organisation&.slug == form.org
```

Comment:
This will always be true if the form does not have an org and and the user does not have an org; which I think means in theory that `can_view_form?` will always be true for any combination of trial user and form created by trial user; which I think means any trial user will be able to view and edit any trial form...